apps/documentation/src/components/sidebar/sidebar.tsx

```diff
@@ -90,6 +90,12 @@ export default function Sidebar({ close }: SidebarProps): JSX.Element {
           onClick={close}
         />
       </SidebarLink>
+      <SidebarLink
+        label="FAQ"
+        href="/documentation/faq"
+        onClick={close}
+        leftSection="❔"
```

Comment:
I wouldn't add here this icon. I think it is better to just keep the one on "Contributing" for better highlighting

Reply:
I understand, maybe we can highlight it more in the future when there is more relevant content